packages/devops/scripts/deployment/deployPackages.sh

```diff
@@ -21,13 +21,9 @@ PACKAGES=("meditrak-server" "web-config-server" "psss-server" "report-server" "w
 # For each package, get the latest and deploy it
 for PACKAGE in ${PACKAGES[@]}; do
     # Set up .env to match the environment variables stored in SSM parameter store
+    yarn download-parameter-store-env-vars --package-name $PACKAGE --environment $ENVIRONMENT
+
     cd ${HOME_DIRECTORY}/packages/$PACKAGE
-    rm .env
-    echo "Checking out ${ENVIRONMENT} environment variables for ${PACKAGE}"
```

Comment:
Perhaps we should still echo this line which can be helpful? Unless it's also done in the other script

Reply:
Yeah the other script does an echo:
`echo "Checking out environment variables under the prefix ${PARAMETER_STORE_PATH_PREFIX}"`

perhaps slightly less readable but I think it's fine 😄